readers/hdfs/src/main/java/com/criteo/hadoop/garmadon/hdfs/HdfsExporter.java

```diff
@@ -217,6 +293,21 @@ private static void printHelp() {
         System.out.println("Usage:");
         System.out.println("\tjava com.criteo.hadoop.garmadon.parquet.HdfsExporter " +
                 "kafka_connection_string kafka_group temp_dir final_dir");
+
+        System.out.println();
+        System.out.println("Properties settable via -D:");
+
+        for (String parameter: Arrays.asList(MESSAGES_BEFORE_EXPIRING_WRITERS, WRITERS_EXPIRATION_DELAY, EXPIRER_PERIOD,
```

Comment:
Maybe you can declare the array along with constants
minor cosmetic note: in a foreach the ':' is detached from any words => `for (String parameter : strArray)`

Reply:
Done